gtk/launcher-specific

```diff
@@ -27,4 +25,10 @@ if [ $needs_update = true ]; then
     $RUNTIME/usr/lib/$ARCH/libgtk2.0-0/gtk-query-immodules-2.0 > $GTK_IM_MODULE_FILE
   fi
 fi
+# create symbolic link to ibus socket path for ibus to look up its socket files
```

Comment:
Wouldn't this better in a different PR?

Reply:
Oh, that shouldn't be part of this PR and should have already been merged in master.